db/migrate/20230117042357_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :post_counter
```

Comment:
- [ ] let's make the post_counter to be plural (posts_counter) as you did with the `comments_counter` and `ikes_counter` This will ensure that you follow the rails naming convention 👍🏻